### PR TITLE
Add a --utility argument to the AppImages.

### LINKED ISSUE #15521: Add a --utility argument for AppImages
The AppImages don't expose their internal filesystem by default, and `--appimage-mount` is a bit to awkward to suggest as a first-class solution.  It would be better if we exposed the utility in the same way we do the dedicated server.

This would also let us rewrite the Resource Center parsers to be based on a directory of AppImages instead of requiring each version to be individually cloned and compiled.

### PR DESCRIPTION
**Pull request by pchote**
Closes #15521.

Adding to the milestone so that we can start depending on it for the resource center as part of the Mod SDK.

### CODE REVIEW

**Review by fusion809 — DISMISSED** [review]
All appears fine. For future reference, I built this PR, applied as a patch to commit 25464 (13f5ef50b93f920230ff56e8ea1b3d7ad3402e4c), and here are the outputs I got from the various AppImages with the `--utility` argument: https://gist.github.com/fusion809/a12d1f773d30df4c08b2fe60a92737b5. Mentioning this log just in case it proves helpful in the future.

File: packaging/linux/buildpackage.sh [hunk 1]
```diff
@@ -120,6 +120,9 @@ build_appimage() {
 	sed "s/{MODID}/${MOD_ID}/g" openra-server.appimage.in > openra-mod-server.temp
 	install -m 0755 openra-mod-server.temp "${APPDIR}/usr/bin/openra-${MOD_ID}-server"
 
+	sed "s/{MODID}/${MOD_ID}/g" openra-utility.appimage.in > openra-mod-utility.temp
+	install -m 0755 openra-mod-utility.temp "${APPDIR}/usr/bin/openra-${MOD_ID}-utility"
```
Comment (obrakmann) [inline packaging/linux/buildpackage.sh]: `openra-mod-utility.temp` needs to be rm'ed at the end of the script.

**Review by obrakmann — DISMISSED** [review]
One minor oversight, works otherwise, :+1:

**Comment by pchote** [comment]
Fixed and rebased.

**Review by obrakmann — APPROVED** [review]
:+1:

**Comment by obrakmann** [comment]
[Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)/_compare/b7d3e3cf64deceb63655d701d438f869a1c859ba)

**Pull request merged**